Resources/Prototypes/Entities/Clothing/Uniforms/base.yml

```diff
@@ -0,0 +1,20 @@
+- type: entity
+  parent: Clothing
+  id: ClothingUniformBase
+  abstract: true
+  components:
+  - type: Sprite
+    state: icon
+  - type: Clothing
+    Slots: [innerclothing]
```

Comment:
Should be new line IMO.

Reply:
I'm assuming you mean 

`Slots:
 - innerclothing`